contractcourt/channel_arbitrator_test.go

```diff
@@ -590,7 +590,7 @@ func TestChannelArbitratorRemoteForceClose(t *testing.T) {
 	chanArb.cfg.ChainEvents.RemoteUnilateralClosure <- &RemoteUnilateralCloseInfo{
 		UnilateralCloseSummary: uniClose,
 		CommitSet: CommitSet{
-			ConfCommitKey: &RemoteHtlcSet,
+			ConfCommitKey: fn.Some(RemoteHtlcSet),
```

Comment:
I'm noticing that there's never a time that this is set to None. Is it every possible for it to be None?

Reply:
Looking at the code, I don't really see a case where we check for it being `nil`, seems like legacy usecase, however its part of the db so removing this structure would require a migration. However it gives us the possibility to know which commitment set confirmed, when querying the db. So let's keep it ?